Makefile

```diff
@@ -1,5 +1,6 @@
 export COMPOSE_PROJECT_NAME=monoetna
 export NODE_ENV=development
+export MONOETNA_SHA=$(shell git rev-parse HEAD) 
```

Comment:
This is basically used in development to allow testing of the git-sha being stored in a figure's `dependencies`